Makefile

```
@@ -90,3 +90,37 @@ test-ci: system-tests unit-tests coverage
 docs:
 	rm -rf docs
 	mkdir docs
+	@-make tracing
+
+tracing: report.lobster
+	mkdir -p docs
+	make lobster/html/assets.py
+	./lobster-html-report report.lobster --out=docs/tracing.html
```

```suggestion
	lobster-html-report report.lobster --out=docs/tracing.html
```
remove "./" on all occurrences. adding the workspace folder to PATH solves this globally